modules/howtos/examples/Import.java

```diff
@@ -0,0 +1,411 @@
+      
+      for (Object row: JsonArray.fromJson(content)) {
+        JsonObject json = ((JsonObject) row);
+        upsertRow(json.toMap());
+      }
+    }
+    catch (java.io.FileNotFoundException e) {
+      System.out.println("handle FileNotFoundException...");
+    }
+    catch (java.io.IOException e) {
+      System.out.println("handle IOException...");
+    }
+  }
+  // end::importJSON[]
+  
+  // tag::importJSON_batch[]
+  public void importJSON_batch() {
+    // tag::omit[]
+    System.out.println("importJSON_batch");
+    // end::omit[]
+
+    try {
+      String content  = new String(
+        Files.readAllBytes(
+          Paths.get("howtos/examples/import.json")),
+        StandardCharsets.UTF_8);
+      
+      Flux<MutationResult> results = 
+        Flux.fromIterable(JsonArray.fromJson(content))
+          .map(row -> ((JsonObject) row).toMap())
+          .map(map -> preprocess(map))
+          .flatMap(doc -> reactiveCollection.upsert(doc.getId(), doc.getContent()));
+          
+      results.subscribe(System.out::println);
+      results.blockLast(Duration.of(60, ChronoUnit.SECONDS));
+    }
+    // ...
+    // tag::omit[]
+    catch (java.io.FileNotFoundException e) {
+      System.out.println("handle FileNotFoundException...");
+    }
+    catch (java.io.IOException e) {
+      System.out.println("handle IOException...");
+    }
+    System.out.println("DONE");
+    // end::omit[]
+  }
+  // end::importJSON_batch[]
+  
+  // tag::importJSONL[]
+  public void importJSONL() {
+    try {
+        BufferedReader br =
+          new BufferedReader(
+            new FileReader("howtos/examples/import.jsonl"));
+        String line;
+        while ((line = br.readLine()) != null) {
+          Map<String,Object> row =
+            JsonObject.fromJson(line).toMap();
+            
+          upsertRow(row);
+      }
+    }
+    // ...
+    // tag::omit[]
+    catch (java.io.FileNotFoundException e) {
+      System.out.println("handle FileNotFoundException...");
+    }
+    catch (java.io.IOException e) {
+      System.out.println("handle IOException...");
+    }
+    // end::omit[]
+  }
+  // end::importJSONL[]
+  
+  // tag::importJSONL_batch[]
+  public void importJSONL_batch() {
+    // tag::omit[]
+    System.out.println("importJSONL_batch");
+    // end::omit[]
+
+    Flux<String> lines = Flux.using(
+      () -> Files.lines(Paths.get("howtos/examples/import.jsonl")),
+      Flux::fromStream,
+      BaseStream::close);
+
+    Flux<MutationResult> results =
+      lines
+          .map(line -> JsonObject.fromJson(line).toMap())
+          .map(map -> preprocess(map))
+          .flatMap(doc -> reactiveCollection.upsert(doc.getId(), doc.getContent()));
+      
+    results.subscribe(System.out::println);
+    results.blockLast(Duration.of(60, ChronoUnit.SECONDS));
+    // tag::omit[]
+    System.out.println("DONE");
+    // end::omit[]
+  }
+  // end::importJSONL_batch[]
+}
+
+// tag::JsonDocument[]
+class JsonDocument {
```

Comment:
Hmm, I think the build is failing due to this class existing in https://github.com/couchbase/docs-sdk-java/blob/release/3.2/modules/hello-world/examples/KvBulkHelloWorld.java 

Because all these classes are part of the same "package", it's still accessible when using `class <Your-Class-Name-Here>`, so it comes up as a duplicate. 
I *think* `private class` could mitigate this, if you want to keep this in here. Or even rename it to something else.

Another option would be to reference the tag in `KvBulkHelloWorld` for the class, but it might look odd to the user if they don't see it when they access the full example on Github.

Reply:
woo, lemme give private a go

Reply:
(ended up nesting the class instead)